src/presence_analyzer/utils.py

```diff
@@ -216,3 +244,17 @@ def work_hours(items):
         start_hours[date.weekday()].append(seconds_since_midnight(start))
         end_hours[date.weekday()].append(seconds_since_midnight(end))
     return (start_hours, end_hours)
+
+
+def total_hours(items, month, year):
+    """
+    Returns total working hours.
+    """
+    seconds = 0
+    if not items:
+        return 0
+    for date in items:
+        if (calendar.month_name[date.month] == month and
+            int(date.year) == int(year)):
```

Comment:
https://www.python.org/dev/peps/pep-0008/#multiline-if-statements
PEP8 says this is a correct indent
`int()` is incorrect?